skfda/exploratory/visualization/_ddplot.py

```diff
@@ -0,0 +1,139 @@
+"""DD-Plot Module.
+
+This module contains the necessary functions to construct the DD-Plot.
+To do this depth is calculated for the two chosen distributions, and then
+a scatter plot is created of this two variables.
+"""
+
+from typing import List, Optional, TypeVar
+
+from matplotlib.axes import Axes
+from matplotlib.figure import Figure
+
+from ...exploratory.depth.multivariate import Depth
+from ._utils import _get_figure_and_axes, _set_figure_layout_for_fdata
+
+T = TypeVar('T')
+S = TypeVar('S', Figure, Axes, List[Axes])
+
+
+class DDPlot:
+    """
+    DDPlot visualization.
+
+    Plot the depth of our fdata elements in two
+    different distributions, one in each axis. It is useful to understand
+    how our data is more related with one subset of data / distribution
+    than another one.
+    Args:
+        fdata: functional data set that we want to examine.
+        dist1: functional data set that represents the first distribution that
+            we want to use to compute the depth (Depth X).
+        dist2: functional data set that represents the second distribution that
+            we want to use to compute the depth (Depth Y).
+        depth_method: method that will be used to compute the depths of the
+            data with respect to the distributions.
+    """
+
+    def __init__(
+        self,
+        fdata: T,
+        dist1: T,
+        dist2: T,
+        depth_method: Depth[T],
+    ) -> None:
+        self.fdata = fdata
+        self.dist1 = dist1
+        self.dist2 = dist2
+        self.depth_method = depth_method
+        self.depth_method.fit(fdata)
+
+    def plot(
+        self,
+        chart: Optional[S] = None,
+        *,
+        fig: Optional[Figure] = None,
+        axes: Optional[List[Axes]] = None,
```

Comment:
The computed depths are always real numbers, independently of the dimensions of the functions. Thus, only one axes is really needed, and `n_rows` and `n_cols` are not necessary.